docs/examples/webextension/background.js

```diff
@@ -0,0 +1,18 @@
+var EXPERIMENT_ID = 'webextension-example-1';
+
+function sendTelemetryPing(data) {
+  document.getElementById('tp-proxy').contentWindow.postMessage({
+    op: 'queueTelemetryPing',
+    data: {
+      subject: EXPERIMENT_ID,
+      data: data
+    }
+  }, '*');
```

Comment:
Yeah, but until/unless I figure out a way to configure this for dev/stage/prod environment it's `*` for now